README.md

```diff
@@ -20,29 +20,23 @@ gnome-abrt is part of [the ABRT project](https://github.com/abrt/).
 
 ### Development dependencies
 
-Build dependencies can be listed by:
+Build dependencies can be installed using:
 
-    $ ./autogen.sh sysdeps
-
-or installed by:
-
-    $ ./autogen.sh sysdeps --install
-
-The dependency installer gets the data from [the rpm spec file](gnome-abrt.spec.in)
+    # dnf builddep --spec gnome-abrt.spec
 
 ### Building from sources
 
 When you have all dependencies installed run the following commands:
 
-    $ ./autogen.sh --prefix=/usr --sysconfdir=/etc --localstatedir=/var --sharedstatedir=/var/lib
-    $ make
+    $ meson build
+    $ ninja -C build
 
 ### Checking
 
 gnome-abrt uses pylint to validate source codes. If pylint prints out any issue,
 the test will fail. Run the test by:
 
-    $ make check
+    $ ninja -C build test
```

Comment:
```
[1/2] Running all tests.
No tests defined.
```

There is also still a `make check` in spec file, and`checkoption` that en(dis)abled pylint.

Reply:
1. Oops.
2. Linting was supposed to be in the description, but I forgot about it while I was typing, so thanks for the reminder.

Reply:
It’s there and it seems like it’s working.